Uplift/Controllers/FilterViewController.swift

```diff
@@ -143,14 +147,22 @@ class FilterViewController: UIViewController, RangeSeekSliderDelegate {
         contentView.addSubview(startTimeClassTypeDivider)
 
         // CLASS TYPE SECTION
-        classTypeDropdown = UITableView(frame: .zero, style: .grouped)
+        classTypeDropdownHeader = DropdownHeaderView(frame: .zero)
+        contentView.addSubview(classTypeDropdownHeader)
+        
+        classTypeDropdownFooter = DropdownFooterView(frame: .zero)
```

Comment:
same here

Reply:
<img width="1059" alt="image" src="https://user-images.githubusercontent.com/43295248/74597411-2011a880-502d-11ea-94de-c09ca21c1383.png">
It won't let me.

Reply:
oh its because the custom init for `DropdownHeaderView` takes in a frame as an argument right now. Since we do everything w/ constraints, you can remove that as an argument and just do `super.init(frame: .zero)` in the `init`